include/effects/Saturation.h

```diff
@@ -65,14 +65,17 @@ namespace openshot
 
 	public:
 		Keyframe saturation;	///< The color saturation: 0.0 = black and white, 1.0 = normal, 2.0 = double saturation
+		Keyframe saturation_R;	///< Red color saturation
+		Keyframe saturation_G;	///< Green color saturation
+		Keyframe saturation_B;	///< Blue color saturation
 
 		/// Blank constructor, useful when using Json to load the effect properties
 		Saturation();
 
 		/// Default constructor, which takes 1 curve, to adjust the color saturation over time.
 		///
 		/// @param new_saturation The curve to adjust the saturation of the frame's image (0.0 = black and white, 1.0 = normal, 2.0 = double saturation)
-		Saturation(Keyframe new_saturation);
+		Saturation(Keyframe new_saturation, Keyframe new_saturation_R, Keyframe new_saturation_G, Keyframe new_saturation_B);
```

Comment:
You didn't start it at all, so it's not your fault, but... very _few_ of our effects use this lame `new_whatever` parameter naming, thankfully, because it's really awful. I'm guessing whoever did it misguidedly wanted to distinguish between the parameter values and the class member, but there's no need for that. My vote would just be for `saturation`, `saturation_R`, `saturation_G`, and `saturation_B` as the parameter names, same as the corresponding class member variables.

Also, you need to add your new parameters to the documentation comment above the declaration.

Reply:
(Actually, the documentation comment needs updating in general, as it no longer "takes 1 curve".)

Reply:
> You didn't start it at all, so it's not your fault, but... very _few_ of our effects use this lame `new_whatever` parameter naming, thankfully, because it's really awful. I'm guessing whoever did it misguidedly wanted to distinguish between the parameter values and the class member, but there's no need for that. My vote would just be for `saturation`, `saturation_R`, `saturation_G`, and `saturation_B` as the parameter names, same as the corresponding class member variables.

Fixed by f9c5ea16236dba6a6bd8843226ed907f7e7da6e5.

Reply:
> Also, you need to add your new parameters to the documentation comment above the declaration.

Fixed by 0d7468b0704df1dc985817c32c47ea9ce0a59e4d.

Reply:
> (Actually, the documentation comment needs updating in general, as it no longer "takes 1 curve".)

Fixed by 88c04944b1d073b3ece17aca3920763cff6edae0.